docs/source/distributions/k8s/chroma-k8s.yaml.template

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: chromadb-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: gp2
```

Comment:
I'd avoid specifying a storageClassName and let the namespace handle the default sc. Not all clusters run on AWS :)